templates/td-agent-bit.conf.j2

```diff
@@ -0,0 +1,45 @@
+[SERVICE]
```

Comment:
Start with commented line `# {{ ansible_managed }}` to inform that this file is managed by ansible.